style.css

```diff
@@ -391,3 +429,256 @@ textarea:hover {
+    align-content: space-between;
+    font-family: 'Inter', sans-serif;
+    font-size: 17px;
+  }
+
+  #envelope {
+    font-size: 21px;
+    padding-right: 8vw;
+  }
+
+  .blackorange {
+    background-image: url("./img/desktop.background.png");
+    background-repeat: no-repeat;
+    background-position: right bottom;
+    background-color: #1c1a19;
+    background-size: 30vw;
+    display: flex;
+    flex-direction: row;
+    justify-content: initial;
+    align-items: center;
+  }
+
+  #texttitle {
+    font-family: 'Crete Round', serif;
+    color: #1c1a19;
+    font-size: 52px;
+    font-weight: 200;
+    padding: 25px 10px 50px 0;
+    border-bottom: #212121 1px solid;
+    margin-bottom: 50px;
+  }
+
+  .icons {
+    order: -1;
+    margin: 0 1vw 0 1vw;
+  }
+
+  #welcometext {
+    width: 40%;
+    margin: 0 1vw 0 1vw;
+  }
+
+  .iconitems {
+    display: list-item;
+  }
+
+  .cards {
+    display: grid;
+    width: 95vw;
+    grid-gap: 20px;
+    align-items: initial;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr;
+    grid-auto-flow: row;
+    padding: 30px;
+    height: auto;
+  }
+
+  .card {
+    height: auto;
+    background: linear-gradient(0deg, rgb(53, 53, 53) 0%, rgba(200, 200, 200, 0.5) 100%) url("./img/img.background.cards.png");
+    background-repeat: no-repeat;
+  }
+
+  .smoke {
+    visibility: show;
+  }
+
+  .card:hover {
+    background: url("./img/img.background.cards.png");
+    color: rgba(255, 255, 255, 0);
+    transition: 0.7s ease-in-out;
+  }
+
+  .card:hover > :nth-child(3) {
+    visibility: hidden;
+    background-color: #000;
+  }
+
+  .card .cardbottom {
+    opacity: 0;
+  }
+
+  .card:hover > .cardbottom {
+    opacity: 100;
+    transition: 0.7s ease-in-out;
+  }
+
+  .mywork {
+    width: 95vw;
+    padding: 15px;
+    height: auto;
+    display: flex;
+    flex-direction: column;
+    align-items: stretch;
+  }
+
+  #imageindex {
+    min-width: 600px;
+    width: 1fr;
+    height: auto;
+    padding-right: 15px;
+  }
+
+  .aboutme {
+    background-image: url("./img/img.background.aboutme.png");
+    background-position: right;
+    background-size: 100vh;
+    justify-content: center;
+    border-bottom: #fff 1px solid;
+    width: 95vw;
+    padding: 15px;
+  }
+
+  .language {
+    display: flex;
+    height: 100vh;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: space-between;
+    width: 93.5vw;
+    padding: 25px;
+  }
+
+  .skillstitle {
+    display: flex;
+    align-items: center;
+    padding: 20px;
+  }
+
+  .skillstitle h2 {
+    font-size: 40px;
+  }
+
+  .skills-img {
+    width: 20px;
+    height: 20px;
+    padding-bottom: 25px;
+  }
+
+  .skills {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+  }
+
+  .skillitem3 li {
+    font-size: 20px;
+  }
+
+  #textaboutme {
+    max-width: 512px;
+  }
+
+  #myworktext {
+    display: flex;
+    flex-direction: column;
+    border: #000;
+  }
```

Comment:
Great job @dgonzalesi 🤝

- [ ] Your card's text seems to overlap the image as the screen size reduces. You may please try to make your design more responsive.
- Your design
![text overlap](https://user-images.githubusercontent.com/63917725/178076621-9ebb9882-af63-4416-a7c9-45813797af5e.PNG)